src/schedule-and-details/utils.js

```diff
@@ -90,8 +91,21 @@ const validateScheduleAndDetails = (courseDetails, intl) => {
   return errors;
 };
 
+const updateWithDefaultValues = (values) => {
+  const { entranceExamMinimumScorePct } = values;
+  if (entranceExamMinimumScorePct === '') {
+    return {
+      ...values,
+      entranceExamMinimumScorePct: defaultEntranceExamMinimumScorePct,
```

Comment:
this will always overwrite `entranceExamMinimumScorePct` value from `...values`, is it expected?